src/main/resources/script_templates/ImageJ2-Scripts/Apply_Threshold.py

```diff
@@ -0,0 +1,18 @@
+# @ImageJ ij
+# @Dataset data
+ 
+method_threshold = "otsu"
+ 
+# Get the histogram
+histo = ij.op().run("image.histogram", data.getImgPlus())
+ 
+# Get the threshold
+threshold = ij.op().run("threshold.%s" % method_threshold, histo)
+ 
+# Apply the threshold
+thresholded = ij.op().run("threshold.apply", data.getImgPlus(), threshold)
+ 
+# Create output
+thresholded = ij.dataset().create(thresholded)
+ 
+ij.ui().show("thresholded", thresholded))
```

Comment:
Hi @hadim 

Otsu threshold could be applied more concisely using

```
thresholded = ops.threshold().otsu(img)
```

However the approach you use would be useful for a little app that allows any threshold method.  I wonder if it is possible to use the 'choice' widget to make a drop down that allow the user to select from a number of threshold methods.